easybuild/easyconfigs/v/VASP/VASP-6.5.0-NVHPC-25.1.eb

```
@@ -0,0 +1,97 @@
+# This build enables both HDF5 and Wannier-90, both are optional but recommended
+# Furthermore, it sets the vdw_kernel as an external file (courtesy of Mikael Öhman)
+# Author: J. Saßmannshausen (Imperial College London/UK)
+
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '6.5.0'
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a
+computer program for atomic scale materials modelling, e.g. electronic
+structure calculations and quantum-mechanical molecular dynamics,
+from first principles.
+
+This build includes HDF5 and the Wannier-90 functions."""
+
+toolchain = {'name': 'NVHPC', 'version': '25.1'}
+
+download_instructions = """VASP is proprietary software, see http://www.vasp.at/index.php/faqs
+on how to obtain a license for the software."""
+sources = [
+    '%(namelower)s.%(version)s.tar',
+    'vdw_kernel.bindat.%(version)s.zip',
+]
+patches = [
+    'VASP-6.5.0_enable_hdf5_wannier90_nvhpc.patch',
+    'VASP-6.5.0_external_vdwkernel.patch',
+]
+checksums = [
+    {'vasp.6.5.0.tar':
+     '87f5a5416633af20fa8ea63f3348db477d8fc378feab4f839a617fc0663875f9'},
+    {'vdw_kernel.bindat.6.5.0.zip':
+     'b2b4bf577e6d4ca1aa6d463cf9325afb834a88099177fb585fe90b0b677f8484'},
+    {'VASP-6.5.0_enable_hdf5_wannier90_nvhpc.patch':
+     '85b0fc899e7896aa5ce1cebcb584b0958892a94a53aa08661f6f203f3bfadb89'},
+    {'VASP-6.5.0_external_vdwkernel.patch':
+     '1533ab6e045e4028f9e7d0de6b026654b9079569be81353bb460991657e48f0c'},
+]
+
+dependencies = [
+    ('FFTW' , '3.3.10'),
```

tiny code style fix:
```suggestion
    ('FFTW', '3.3.10'),